apps/ewallet_db/priv/repo/migrations/20180704095243_update_unique_index_on_exchange_pair.exs

```diff
@@ -0,0 +1,24 @@
+defmodule EWalletDB.Repo.Migrations.UpdateUniqueIndexOnExchangePair do
+  use Ecto.Migration
+
+  def up do
+    execute ~s"DROP INDEX exchange_pair_from_token_uuid_to_token_uuid_deleted_at_index"
+
+    execute ~s"CREATE UNIQUE INDEX exchange_pair_from_token_uuid_to_token_uuid_deleted_at_not_null
```

Comment:
You've given up on Ecto? 😅

Reply:
Oh Ecto does support [partial indexes](https://hexdocs.pm/ecto/Ecto.Migration.html#index/3-partial-indexes)! Updating...